modules/security-context-constraints-about.adoc

```diff
@@ -124,6 +125,12 @@ The `restricted` SCC:
 * Requires that a pod is run with a pre-allocated MCS label
 * Allows pods to use any FSGroup
 * Allows pods to use any supplemental group
+
+[NOTE]
+====
+The `restricted` SCC still configures `allowPrivilegeEscalation: true`, which disables the `no_new_privs` protection flag. If you want an additional protection from disabling `no_new_privs`, you can create a custom SCC with the flag disabled and configure the access to only appropriate users in place of `restricted`. For more details about flag, see link:https://www.kernel.org/doc/html/latest/userspace-api/no_new_privs.html[The Linux Kernel documentation].
```

Comment:
I'm not confident I understand this. Particularly the second sentence. Perhaps this is worth making clearer?

If you want an additional protection from disabling `no_new_privs`,(meaning: if you want most of what you get in `restricted`, but with `no_new_privs` enabled?) you can create a custom SCC with the flag (which flag?) disabled and configure the access to only appropriate users _in place of `restricted`_(what is `restricted` in place of? users?)

Reply:
Perhaps something like this:

````
[NOTE]
====
In the `restricted` SCC, `allowPrivilegeEscalation` is set to `true`, which disables the `no_new_privs` protection flag. If you want to enable `no_new_privs` and use an SCC that is similar to the `restricted` SCC, you can create a custom SCC that uses the same configuration but with `allowPrivilegeEscalation` is set to `true`.
````
Since the note isn't about `no_new_privs`, but rather about how to enable it with a restricted security context, I suggest we leave out the external link.

Reply:
@dbaker-rh This version is clearer, but may be inaccurate. Can you confirm this is right? Thanks!